code/game/objects/items/devices/scanners.dm

```diff
@@ -165,17 +165,17 @@ GENE SCANNER
 		mob_status = span_alert("<b>Deceased</b>")
 		oxy_loss = max(rand(1, 40), oxy_loss, (300 - (tox_loss + fire_loss + brute_loss))) // Random oxygen loss
 
+	render_list += "[span_info("Analyzing results for [M]:")]\n<span class='info ml-1'>Overall status: [mob_status]</span>\n"
+	
+	SEND_SIGNAL(M, COMSIG_LIVING_HEALTHSCAN, render_list, advanced, user, mode)
```

Comment:
while you're here, go to the top of this proc and rename the var `M`, it's not a descriptive var